workflow-engine/src/main/java/com/redhat/parodos/workflows/workflow/ParallelFlowReport.java

```diff
@@ -139,4 +139,9 @@ public WorkContext getWorkContext() {
 		return workContext;
 	}
 
+	@Override
+	public String getAlertMessage() {
+		return null;
```

Comment:
Perhaps this needs to be replaced with logic similar to 
```
	/**
	 * Return the first error of partial reports.
	 * @return the first error of partial reports.
	 */
	@Override
	public Throwable getError() {
		for (WorkReport report : reports) {
			Throwable error = report.getError();
			if (error != null) {
				return error;
			}
		}
		return null;
	}
```

Reply:
Addressed, but I think it's better to return the last inserted alertMessage. Can you please take a look ?